helm/charts/kratos/values.yaml

```diff
@@ -267,6 +267,14 @@ securityContext:
   allowPrivilegeEscalation: false
   privileged: false
 
+# Horizontal pod autoscaling configuration
+autoscaling:
```

Comment:
In the issue you've mentioned kratos is not impacted however I see you did the same change as in hydra?
Also with hydra by default we give HPA with max to 3 replicas (we don't have demo profiles etc) so maybe let's limit a bit 100 replicas ;-)

Reply:
That seems reasonable. I think Keto uses 100 as its maximum, so you might want to look at changing that as well.